docs/config/README.md

```diff
@@ -1860,6 +1860,7 @@ You can disable the module or use the `windows_starship` option to use a Windows
 | `staged`            | `'+'`                                         | The format of `staged`                                                                                      |
 | `renamed`           | `'»'`                                         | The format of `renamed`                                                                                     |
 | `deleted`           | `'✘'`                                         | The format of `deleted`                                                                                     |
+| `typechanged`       | `""`                                          | The format of `typechange`                                                                                  |
```

Comment:
So typechanges aren't shown by default? That's a bit of a shame to me, since the reason I reported it was I found it surprising that starship wasn't summarizing something that I had to commit by default.

(It also looks like this would be better off using a single-quoted string for consistency)